sccm/core/clients/manage/co-management-prepare.md

```diff
@@ -54,9 +54,12 @@ For example, if you had the following values:
   > [!Note]    
   > Use the **IdentifierUri** value found in the **vSMS_AAD_Application_Ex** SQL view for the **AAD Resource ID URI** value.
 
+> [!IMPORTANT]
+> If you are using internal CA and didn’t publish your CRL to internet, you need to add /nocrlcheck CCMHTTPSSTATE=31 in client installation command line.
```

Comment:
Let's keep this alert, but I recommend a few slight changes: "If you're using an internal certificate authority and didn't publish your certificate revocation list (CRL) to the internet, you need to add **/NoCRLCheck CCMHTTPSSTATE=31** to the client installation command line."